src/lib/storage/frame_of_reference_column.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include <boost/hana/contains.hpp>
+#include <boost/hana/tuple.hpp>
+#include <boost/hana/type.hpp>
+
+#include <type_traits>
+
+#include <array>
+#include <memory>
+
+#include "base_encoded_column.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+class BaseCompressedVector;
+
+/**
+ * @brief Column implementing frame-of-reference encoding
```

Comment:
Easy to google, a link to an explanation about FOR would be cool

Reply:
Is the comment below enough?

Reply:
Now that I know what FOR is, yes. If it would have been enough before, I don't know. Your choice.

Reply:
I’ll add a link to my thesis to all encoding schemes once I have handed it in :P